.pre-commit-config.yaml

```diff
@@ -971,16 +971,16 @@ repos:
   # <---- Changelog --------------------------------------------------------------------------------------------------
 
   # ----- Invoke ---------------------------------------------------------------------------------------------------->
-      - id: pip-tools-compile
-        alias: compile-ci-invoke-3.6-requirements
-        name: Linux CI Py3.6 Invoke Requirements
-        files: ^requirements/static/ci/(invoke\.in|py3.6/(invoke|linux)\.txt)$
-        pass_filenames: false
-        args:
-          - -v
-          - --py-version=3.6
-          - --pip-args=--constraint=requirements/static/ci/py{py_version}/linux.txt
-          - requirements/static/ci/invoke.in
+##      - id: pip-tools-compile
```

Comment:
Why are we commenting it out, why note just remove it?

Reply:
Commented it out in case, it was a no-go, and didn't want to have to retype all that back in.
During testing I had removed it, but then put it back to try and work around the Py 3.6 pre-commit not recognizing packaging to install even though it found packaging==3.0.9.

Given your comment about running old tests https://saltstack.slack.com/archives/C02D6MGTGF3/p1671200671793249?thread_ts=1671162285.865489&cid=C02D6MGTGF3
Decided to just leave it commented out rather than remove it as part of the final cleanup, in case it had to be added back in from the review.